server/apps/immich/src/api-v1/job/dto/create-job.dto.ts

```diff
@@ -0,0 +1,15 @@
+import { QueueNameEnum } from '@app/job';
+import { ApiProperty } from '@nestjs/swagger';
+import { IsEnum, IsNotEmpty } from 'class-validator';
+
+export class CreateJobDto {
```

Comment:
As far as I can tell this DTO isn't used anywhere. What do we need it for?

Reply:
Ah we don't need this anymore, it was there before Zack's suggestion to change the schema